lib/rules/web/admin_console/4.0/secrets.xyaml

```diff
@@ -40,8 +40,8 @@ create_generic_secret:
     if_param: key 
     ref: set_pair_key
   action:
-    if_param: value
-    ref: set_pair_value
+    if_param: textarea_value
+    ref: set_textarea_value
```

Comment:
To not break your tests for `OCP-20800`, you also need update `set_textarea_value` rules to always send keys to the last `textarea`

Reply:
@yapei Never mind, I checked the features and OCP-20800 is the only case needs update, also there is another rule `click_one_dropdown_action` updating a param. So soner or later the OCP-20800 needs to patch a fix PR, so I update this one together this time, don't worry there's no big change.

Reply:
ok for me

Reply:
@yapei so you want PR merged?

Reply:
@akostadinov  no problem, let's merge it